core/primitives/src/runtime/config_store.rs

```diff
@@ -0,0 +1,147 @@
+use crate::runtime::config::RuntimeConfig;
+use crate::types::ProtocolVersion;
+use std::collections::BTreeMap;
+use std::iter::FromIterator;
+use std::ops::Bound;
+use std::sync::Arc;
+
+macro_rules! include_config {
+    ($file:expr) => {
+        include_bytes!(concat!("../../../../nearcore/res/runtime_configs/", $file))
+    };
+}
+
+/// Stores pairs of protocol versions for which runtime config was updated and
+/// the new runtime config in bytes.
+/// Protocol versions are given in increasing order. First one is always 0, so that each version is
+/// mapped to some config.
+static CONFIGS: [(ProtocolVersion, &[u8]); 2] =
+    [(0, include_config!("29.json")), (42, include_config!("42.json"))];
```

Comment:
Yeah, this is ineed much, much better than pulling some random crate.io crate for this!